cmake/dependencies.cmake

```diff
@@ -177,16 +177,18 @@ function(ov_download_tbbbind_2_5)
 
     if(WIN32 AND X86_64)
         RESOLVE_DEPENDENCY(TBBBIND_2_5
-                ARCHIVE_WIN "tbbbind_2_5_static_win_v1.zip"
+                ARCHIVE_WIN "tbbbind_2_5_static_win_v2.zip"
                 TARGET_PATH "${TEMP}/tbbbind_2_5"
                 ENVIRONMENT "TBBBIND_2_5_ROOT"
-                SHA256 "a67afeea8cf194f97968c800dab5b5459972908295242e282045d6b8953573c1")
+                SHA256 "6e764e06d75f52c7d775ccd8c81d4472f4084b27906e6213b93df39f728b6e64"
+                USE_NEW_LOCATION TRUE)
     elseif(LINUX AND X86_64)
         RESOLVE_DEPENDENCY(TBBBIND_2_5
-                ARCHIVE_LIN "tbbbind_2_5_static_lin_v2.tgz"
+                ARCHIVE_LIN "tbbbind_2_5_static_lin_20230112_v3.1.tgz"
```

Comment:
Please remove data in file name

Reply:
BTW, is it really TBBBind 2.5 or a newer version?

Reply:
> BTW, is it really TBBBind 2.5 or a newer version?

I think the version of TBBBind represents the version of the linked hwloc version. And "tbbbind_2_5" links with HWLOC 2.5 and later according to the documentation of oneTBB.